bitnami/owncloud/values.yaml

```diff
@@ -217,6 +221,23 @@ persistence:
 
     accessMode: ReadWriteOnce
     size: 8Gi
+    # extraVolumes:
+    #   - name: shared-data
+    #     persistentVolumeClaim:
+    #       claimName: shared-data-pvc
+
+    # extraVolumeMounts:
+    #   - name: shared-data
+    #     mountPath: /mnt/data/files/admin/files/shared-data
```

Comment:
I wouldn't include these parameters under the "persistence" section since volumes can be used for other stuff (e.g. mounting a ConfigMap as a volume). You can use RabbitMQ as a reference to see what we do in other charts, see:

- https://github.com/bitnami/charts/blob/master/bitnami/rabbitmq/values.yaml#L262

Reply:
I personally think it'd be nice to have a high-level `owncloud` value that indicates the changes are made to the main owncloud container. This would both be in line with how [nextcloud chart](https://github.com/helm/charts/tree/master/stable/nextcloud#installing-the-chart) evolved, keeping the two more in sync, and personally think it makes it easier to understand.
This would mean having `owncloud.extraEnv` and `owncloud.extraVolumes` which seems better to me, but also it's ultimately just a naming issue so I am not at all set in stone in that opinion if you have other reasons like internal consistency for preferring to not nest things under `owncloud`

Reply:
P.S. I definitely think highest scope is better than inside persistence though if `owncloud` value is not to be added. The choice to put it in persistence was mostly because some other chart developers are more inclined to accept nested than high-level values and wanted this to not look too intrusive :)

Reply:
We usually have a high-level value (such as `owncloud`, `mariadb`, etc.) when the chart deploys several components, so we can differentiate between what parameters affect componentA or componentB (e.g see [Thanos](https://github.com/bitnami/charts/tree/master/bitnami/thanos)). It's not vey common to do so on charts such as OwnCloud that only one component (plus MariaDB which is managed by a different chart).
Regarding including these parameters in a higher scope than `persistence`, as I said before volumes can be used for different purposes and that's why I think it should be treated as different parameters.